app/models/stripe_record.rb

```diff
@@ -27,7 +27,7 @@ class StripeRecord < ApplicationRecord
     legacy_payment_intent_registered: "payment_intent_registered",
     legacy_success: "success",
     legacy_failure: "failure",
-  }
+  }, prefix: true
```

Comment:
Have you made certain that this doesn't break other places in the code? From the top of my head, I am mostly concerned about `ConnectedStripeAccount` and `PaymentIntent` models

Reply:
Removed this, I didn't realize that it would cause the non-prefixed methods to no longer work (but it makes sense)